tests/components/orderentry/addForm/DraftDataTable.test.jsx

```diff
@@ -66,10 +66,28 @@ describe('Draft Data Table', () => {
     expect(props.handleDiscardOneOrder.mock.calls.length).toEqual(0);
     expect(wrapper).toMatchSnapshot()
   });
+
+  it('should truncate text longer than 155 characters', () => {
+    props = {
+      ...props,
+      draftOrders: [
+        {
+          action: 'DISCONTINUE',
+          drugName: '',
+          drugInstructions: 'Take 2 tablets, with warm water or lime in the morning (before meal) and evening (after meal). Not to be taken with alcohol neither should ne taken by before or while driving',
```

Comment:
You seem to be a fun of beer. 😊

Reply:
Lols. I don't drink, unless it is water

Reply:
That's the way drug prescriptions are usually written on leaflets here in Nigeria. You see words like `not to be be taken during pregnancy, not to be taken with alcohol, and not to be taken when driving`. lols